app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -170,7 +170,14 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         }
     }
 
-    private fun setScreenSize(isTablet: Boolean) {
+    override fun onUpdateToolbarForConfigurationChange(toolbar: BrowserToolbarView) {
+        super.onUpdateToolbarForConfigurationChange(toolbar)
+
+        setToolbarActionsForSize(isTablet = resources.getBoolean(R.bool.tablet))
+    }
+
+    @VisibleForTesting
+    internal fun setToolbarActionsForSize(isTablet: Boolean) {
```

Comment:
```suggestion
    internal fun updateToolbarActions(isTablet: Boolean) {
```
I am wondering if this would be a better function name since we aren't actually setting any value within the function.

Reply:
agreed, will update to `updateToolbarActions`